bin/utils.js

```diff
@@ -4,9 +4,10 @@ const { EOL } = require('os');
 const { readInput } = require('../lib/fileHandler');
 
 function parseFilePaths(argv) {
-  if (argv._.length !== 2) return {};
-  const [inputFilePath, outputFilePath] = argv._;
-  return { inputFilePath, outputFilePath };
+  const errMsg = 'Missing arguments. Please provide both input and output paths.';
```

Comment:
Since this is a PR for adding a default output path option, should this message stay the same as before?